SU2_CFD/src/output/CNEMOCompOutput.cpp

```diff
@@ -559,7 +559,7 @@ void CNEMOCompOutput::LoadVolumeData(CConfig *config, CGeometry *geometry, CSolv
 
 void CNEMOCompOutput::LoadSurfaceData(CConfig *config, CGeometry *geometry, CSolver **solver, unsigned long iPoint, unsigned short iMarker, unsigned long iVertex){
 
-  if ((config->GetKind_Solver() == NEMO_NAVIER_STOKES)) {
+  if (config->GetKind_Solver() == NEMO_NAVIER_STOKES && config->GetViscous_Wall(iMarker)) {
```

Comment:
I would also remove the check for NEMO_NAVIER_STOKES... This kind of logic causes 90% of the bugs in SU2 every time a new options is added something breaks, the simpler the logic, the easier it is to maintain the code.